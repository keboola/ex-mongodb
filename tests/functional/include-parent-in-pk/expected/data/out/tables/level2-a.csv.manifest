{"incremental":false,"columns":["count","itemId","level1_pk"],"column_metadata":{"count":[{"key":"KBC.datatype.nullable","value":true}],"itemId":[{"key":"KBC.datatype.nullable","value":true}],"level1_pk":[{"key":"KBC.datatype.nullable","value":true}]}}