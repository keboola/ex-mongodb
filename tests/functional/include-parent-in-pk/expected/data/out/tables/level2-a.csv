"count","itemId","level1_pk"
"1","123","ad1c2165423aa6c30ae7b2515625a1ab"
"1","123","cf90539a89e1696793d8f8c29b89ca61"
"20","456","77d54caf02384eea2f166dff9cdb70c1"
"20","456","854dab3d9cfa9d5994da2ddeec1e9446"
